### PR TITLE
Some comments for the code and the documents.

### PR DESCRIPTION
**Pull request by augustzms**
Ignore the ones if you think they are not reseaonable.

### CODE REVIEW

File: chain.go [hunk 1]
```diff
@@ -36,14 +37,15 @@ func (c *Chain) TxSearch(query string, prove bool, page, perPage *int, orderBy s
...
-	err := c.c.get(urlChainTxSearch, req, r)
+	err := c.cl.get(urlChainTxSearch, req, r)
 	if err != nil {
 		return nil, err
 	}
 	return r, nil
 }
 
 // BlockSearch search for blocks by BeginBlock and EndBlock events.
+// what are BeginBlock and EndBlock page, perPage.
```
Comment (TheWinds) [inline chain.go:48]: > https://docs.tendermint.com/v0.35/rpc/#/Info/block_search

This API seems to have some problems, I will change it later

File: doc.md [hunk 2]
```diff
@@ -208,23 +216,23 @@ type Cluster struct {
...
+RemoveValidator removes the given validator from Glitter cluster\.
 
 ### func \(\*Cluster\) UpdateValidator
 
 ```go
 func (c *Cluster) UpdateValidator(pubKey string, power int) error
 ```
 
-UpdateValidator update validator to glitter cluster
+UpdateValidator updates validator in Glitter cluster. which will be updated? power?
```
Comment (TheWinds) [inline doc.md:235]: Update validator's power,if the power is zero remove it.

File: doc.md [hunk 3]
```diff
@@ -295,20 +303,24 @@ type Document []byte
 func (m Document) MarshalJSON() ([]byte, error)
 ```
 
-MarshalJSON returns m as the JSON encoding of m\.
+MarshalJSON returns m as the JSON encoding of m\. what is m? should this as d? make sure names are unique for one type and consistant.
```
Comment (TheWinds) [inline doc.md:306]: LGTM,it's a typo

File: cluster.go [hunk 4]
```diff
@@ -16,6 +16,7 @@ type Cluster struct {
 }
 
 // UpdateValidator update validator to glitter cluster
+// any suggestion clu for cluster?
```
Comment (TheWinds) [inline cluster.go:19]: I think it might be better to have the same style

**Review by TheWinds — CHANGES_REQUESTED** [review]
Thanks for your pointers

File: chain.go [hunk 5]
```diff
@@ -7,21 +7,22 @@ import (
...
+	err := c.client.get(urlChainStatus, req, r)
 	if err != nil {
 		return nil, err
 	}
 	return r, nil
 }
 
-// TxSearch Search for transactions
+// TxSearch searches transactions with the given query.
+// what is prove, page and perPage?
```
[the remark this answers is not in the document]
Comment (augustzms) [reply chain.go:25]: 1. Can we add this prove information into the comments as well?
2. For the page/perPage, do you think either add comments explaining would be helpful/we change the name here?

**Pull request merged**